README.rst

```diff
@@ -83,16 +83,33 @@ The configuration settings (with one example endpoint) are as below:
     user: "crate"             # Username to use (default: "crate")
     password: ""              # Password to use (default: "").
     schema: ""                # Schema to use (default: "").
-    max_connections: 5        # The maximum number of concurrent connections (default: 5).
+    max_connections: 0        # The maximum number of concurrent connections (default: runtime.NumCPU()).
                               # It will get forwarded to pgx's `pool_max_conns`, and determines
-                              # the maximum number of connections in the connection pool.
+                              # the maximum number of connections in the connection pool for
+                              # both connection pools (read and write).
+    read_pool_size_max: 0     # Configure the maximum pool size for read operations individually.
+                              # (default: runtime.NumCPU())
+    write_pool_size_max: 0    # Configure the maximum pool size for write operations individually.
+                              # (default: runtime.NumCPU())
     connect_timeout: 10       # TCP connect timeout (seconds) (default: 10).
                               # It has the same meaning as libpq's `connect_timeout`.
     read_timeout: 5           # Query context timeout for read queries (seconds) (default: 5).
     write_timeout: 5          # Query context timeout for write queries (seconds) (default: 5).
     enable_tls: false         # Whether to connect using TLS (default: false).
     allow_insecure_tls: false # Whether to allow insecure / invalid TLS certificates (default: false).
 
+Performance Settings
+--------------------
+
+The service uses two connection pools to the database, one of each for read vs. write
+operations. The configuration settings ``max_connections``, ``read_pool_size_max``, and
+``write_pool_size_max`` essentially configure the maximum connection pool sizes, either
+for both pools at once, or individually.
+
+By default, when not configured otherwise, ``pgx`` configures the maximum pool size
```

Comment:
```suggestion
By default, when not configured otherwise, by either omitting the settings altogether, or using `0` values, ``pgx`` configures the maximum pool size
```

Reply:
Fixed.